src/cartesian/CartesianGrid.js

```diff
@@ -131,6 +136,81 @@ class CartesianGrid extends Component {
     return <g className="recharts-cartesian-grid-vertical">{items}</g>;
   }
 
+  /**
+   * Draw vertical grid stripes filled by colors
+   * @param {Array} verticalPoints either passed in as props or generated from function
+   * @return {Group} Vertical stripes
+   */
+  renderVerticalStripes(verticalPoints) {
+    const { verticalFill } = this.props;
+    if (!verticalFill || !verticalFill.length) { return null; }
+
+    const { fillOpacityVertical, x, y, width, height } = this.props;
```

Comment:
The props `fillOpacityVertical` seems to be very strange.

Reply:
What do mean by strange? You think is it better to use the default FillOpacity property both for horizontal and vertical lines?

Reply:
@costagolub How about use the default fillOpacity?